tools/shared.py

```diff
@@ -2335,11 +2335,11 @@ def metadce(js_file, wasm_file, minify_whitespace, debug_info):
     cmd = [os.path.join(Building.get_binaryen_bin(), 'wasm-metadce'), '--graph-file=' + temp, wasm_file, '-o', wasm_file]
     if debug_info:
       cmd += ['-g']
-    out = subprocess.check_output(cmd)
+    out = run_process(cmd, stdout=PIPE).stdout
     # find the unused things in js
     unused = []
     PREFIX = 'unused: '
-    for line in out.split(os.linesep):
+    for line in out.split('\n'):
```

Comment:
what's this change? don't we need to support `\r\n` on windows?

Reply:
Because our `run_process` has `universal_newlines=True` by default which converts `\r\n` to `\n` automatically. Thus we can safely (and must) use `\n` with `run_process`.

Reply:
Got it, thanks!